helper/database.js

```diff
@@ -0,0 +1,24 @@
+const blueBird = require('bluebird');
```

Comment:
If this is the API for the mongodb-service, it should reside in /persistence or /services or something similar, rather than /helper